content/api/editor-support.md

```diff
@@ -27,5 +21,16 @@ When we release new updates, you'll need to update the locally installed version
 
 ```bash
 pip install git+https://github.com/sourcebots/robot-api --update
-pip install git+https://github.com/sourcebots/robot-api --user --update
 ```
+
+## Permission error
+The above commands may require administrator access to the computer. If you're getting a _Permission Denied_ error of some sorts, you may need to add `--user` to the end. This is likely needed on a shared computer, such as those you may find in your colleges. 
+
+```bash
+pip install git+https://github.com/sourcebots/robot-api --user
+pip install git+https://github.com/sourcebots/robot-api --update --user
+```
+
+{{% notice note %}}
+On macOS / Linux, you may also use `sudo` to run the command as root. On Windows, you may need to re-run using [command prompt as an administrator](https://technet.microsoft.com/en-us/library/cc947813.aspx).
```

Comment:
Minor: I'd be tempted to say "you could also" (or "you can also") rather than "you may also".